index.js

```diff
@@ -73,6 +73,10 @@ class ReactNativeUA {
         bridge.removeTag(tag);
     }
 
+    static setNamedUserId (nameUserId) {
```

Comment:
we need to normalize the methods/variables declarations; variables/methods: snake_case_, classes: CamelCase